rclpy/src/rclpy/node.cpp

```diff
@@ -478,6 +477,21 @@ Node::Node(
   if (RCL_RET_OK != ret) {
     throw RCLError("error creating node");
   }
+
+  if (rcl_logging_rosout_enabled() && enable_rosout) {
+    rclpy::LoggingGuard scoped_logging_guard;
+    ret = rcl_logging_rosout_init_publisher_for_node(rcl_node_.get());
+    if (ret != RCL_RET_OK) {
+      if (rcl_logging_rosout_fini_publisher_for_node(rcl_node_.get()) != RCL_RET_OK) {
+        // Warning should use line number of the current stack frame
+        int stack_level = 1;
+        PyErr_WarnFormat(
+          PyExc_RuntimeWarning, stack_level, "Error in destruction of rosout publisher: %s",
+          rcl_get_error_string().str);
```

Comment:
I think [this comment](https://github.com/ros2/rclpy/blob/1a5e5f0166aa788bf0bad7f394e7fb462e81f442/rclpy/src/rclpy/node.cpp#L486) means there should add line number below.
```suggestion
        PyErr_WarnFormat(
          PyExc_RuntimeWarning, stack_level, "[rclpy| %s : %s ]: error in destruction of rosout publisher: %s",
          RCUTILS_STRINGIFY(__FILE__), RCUTILS_STRINGIFY(__LINE__), rcl_get_error_string().str);
```

Reply:
addressed by https://github.com/ros2/rclpy/pull/1121/commits/a66cc85506a0c36c379fbe52247868523ea844da